qemu/target-arm/unicorn_aarch64.c

```diff
@@ -79,6 +79,15 @@ int arm64_reg_read(struct uc_struct *uc, unsigned int *regs, void **vals, int co
                 case UC_ARM64_REG_CPACR_EL1:
                     *(uint32_t *)value = ARM_CPU(uc, mycpu)->env.cp15.c1_coproc;
                     break;
+                case UC_ARM64_REG_TPIDR_EL0:
+                        *(int64_t *)value = ARM_CPU(uc, mycpu)->env.cp15.tpidr_el0;
```

Comment:
They align with all the blocks except for the UC_ARM64_REG_CPACR_EL1, so I'm thinking it's the UC_ARM64_REG_CPACR_EL1 block that's misaligned. It does seem a little strange to me that all the other blocks are indented 9 spaces, given that the rest of the file is indented 4 spaces